test/webpack/iconv-lite-tests.js

```diff
@@ -11,3 +11,4 @@ require("../sbcs-test");
 require("../turkish-test");
 require("../utf16-test");
 require("../utils-test");
+require("../shiftjis-test");
```

Comment:
I'm wondering if we can add `test/big5-test.js` here too?
For `gbk-test.js` I assume you skipped it because it reads a file, maybe we can inline it or convert it to a JSON file with a single string, so that it can be require()-d?

Reply:
> I'm wondering if we can add test/big5-test.js here too?

Unfortunately `big5-test.js` has one test which uses `iconv`. It's the 3rd one.

> For gbk-test.js I assume you skipped it because it reads a file, maybe we can inline it or convert it to a JSON file with a single string, so that it can be require()-d?

Yeah, sounds like a great idea. Let me see what I can do 😄

Reply:
@ashtuchkin I've noticed you added some more utils and a generation step, which helped me convert both files in the end. Not sure about the whole "fixtures" and if this is how you would like it to be, so looking forward for your feedback.

Reply:
Yep I think it's a great idea. Left a couple of nits and we're ready to merge.